Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -53,6 +56,29 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
                let jsonData = try? JSONSerialization.data(withJSONObject: object, options: [.prettyPrinted]) {
                 outputText = String(data: jsonData, encoding: .utf8)!
             }
+        case .gpx:
+            var gpxText: String = String("<?xml version=\"1.0\" encoding=\"UTF-8\"?>")
+            gpxText.append("\n<gpx xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://www.topografix.com/GPX/1/1\" xsi:schemaLocation=\"http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd\" version=\"1.1\">")
+            
+            guard let routeResponse = routeResponse else { return }
+            guard let routes = routeResponse.routes else { return }
+            
+            let dateFormatter = DateFormatter()
```

Comment:
Can `ISO8601DateFormatter` be used here?

Reply:
What would be the benefit of using `ISO8601DateFormatter` over `DateFormatter`?

Reply:
You don't have to provide a format string as what you have is actually an ISO8601 format.

Reply:
I'll switch to `ISO8601DateFormatter` then. Thanks for the recommendation